doc/configuring.rst

```diff
@@ -268,18 +271,20 @@ directory
 .. _critical_step:
 
 critical
-    Basic usage is adding ``critical=True`` to configuration description.
+    Basic usage is adding ``critical=True`` to `build step`_ description.
     This parameter is used in case of a linear step execution, when the result of some step is
-    critical for the subsequent step execution. If some configuration has `critical` key set to `True`
-    and executing this step fails, no more configurations will be executed during this run.
+    critical for the subsequent step execution. If some step has `critical` key set to `True`
+    and executing this step fails, no more steps will be executed during this run.
     However, all already started :ref:`background <background_step>` steps will be finished
     regardless critical step results.
 
+.. Add proper description of nested steps `critical` functioning somewhere somehow
```

Comment:
Is there issue for this?

Reply:
No. Do we really need it? It's just a part of general documentation update, we have a lot of missing descriptions like this...

Reply:
Yeah, but there is a TODO in the code.
I prefer issue.

Reply:
#535